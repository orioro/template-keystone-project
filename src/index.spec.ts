import { bibliotecaDataApi } from './'
import * as PUBLIC_API from './'
import { testCases } from '@orioro/jest-util'

test('public api', () => {
  expect(Object.keys(PUBLIC_API)).toMatchSnapshot()
})

describe('tests', () => {
  testCases([['parameter', 'output "parameter"']], bibliotecaDataApi)
})
