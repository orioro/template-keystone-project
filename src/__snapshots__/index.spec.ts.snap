// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`public api 1`] = `
Array [
  "bibliotecaDataApi",
]
`;
